src/openrct2/FileClassifier.cpp

```diff
@@ -212,7 +212,7 @@ FileExtension GetFileExtensionType(u8string_view path)
     if (String::iequals(extension, ".sc6"))
         return FileExtension::SC6;
     if (String::iequals(extension, ".sea"))
-        return FileExtension::SC6;
+        return FileExtension::SEA;
```

Comment:
This change is a left-over from the RCT Classic source/origin tab, should I revert it or should I keep it?